src/server.coffee

```diff
@@ -43,7 +40,7 @@ if Meteor.isServer
       @stopped = true
 
       # No client mutators allowed
-      share.JobCollectionBase.__super__.deny.bind(@)
+      Meteor.Collection.prototype.deny.bind(@)
```

Comment:
I had to hardcode this because I have no idea how to get to `__super__` equivalent in new CoffeeScript/JavaScript.

Reply:
See: https://github.com/jashkenas/coffeescript/issues/4858